### ISSUE TITLE
[Feature Request]: Support aarch64 python wheel for Linux

### ISSUE THREAD

**Issue by bonlime** [openvinotoolkit/openvino#19018]
### Request Description

Currently there are builds for arm64, and aarch64 + linux (#15865) , but there is no support for aarch64 + mac. The reason why I need this is because I'm launching a Docker on M1 Mac, and the docker specifies the arch as `aarch64`, it's not a problem for the rest of my packages, they install without problems, but `openvino` is unable to find suitable wheels. 

I believe this has something to do with specified architectures in the published wheels, no code changes other than releasing +1 wheel would be required. I'm not really familiar with how pip detects architectures, but when I run `pip install onnxruntime` on my M1 machine it installs version: `onnxruntime-1.15.1-cp310-cp310-macosx_11_0_arm64.whl` and when I run the same command inside my Docker with aarch64 it installs `onnxruntime-1.15.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl`. Maybe you could have some wheel similar to this for openvino as well? 

### Issue submission checklist

- [X] The feature request or improvement must be related to OpenVINO

**Comment by ilya-lavrenov** [comment]
Hi @bonlime 
We don't provide prebuilt Linux aarch64 wheels because of internal infrastructure issues (but we are in progress of resolving them), but technically it's quite easy to natively build OpenVINO as a wheel package on macOS via docker Linux aarch64 environment. 
If this way works for you, you can follow this instruction https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_linux.md and perform all these steps to install dependencies and build OpenVINO wheel package.

**Comment by ilya-lavrenov** [comment]
Alternatively, you can use conda packages:
```
conda install -c "conda-forge/label/openvino_dev" openvino
```
Currently, users have to use `openvino_dev` label for pre-release packages, but they will be fully available in upcoming 2023.1 release.

Note, if you install OpenVINO using `conda`, the `pip` will "see" this package:
```
conda install -c "conda-forge/label/openvino_dev" openvino
pip list | grep openvino
```
Will print `openvino`

**Comment by bonlime** [comment]
@ilya-lavrenov thanks for a quick response. I'll try both approaches and will see which one is easier for me, I hope this would resolve my question, I'll close this issue in a few days if everything works

**Comment by ilya-lavrenov** [comment]
We have released OpenVINO 2023.1.0 and published wheel packages for arm64 Linux https://pypi.org/project/openvino/#files
Please, try:
```sh
python3 -m pip install openvino
```

**Comment by bonlime** [comment]
@ilya-lavrenov that works, thanks a lot!
